### PR TITLE
Fix #1556

### LINKED ISSUE #1556: A wrong VCDM 2.0 context URL
Hi,
a wrong, non-existent,  context is used at three places. I guess it's typo.

`https://www.w3.org/2018/credentials/v2` 

please see:
 * [Example 30](https://w3c.github.io/vc-data-model/#example-verifiable-credential-using-the-data-integrity-bbs-cryptosuite-with-a-base-proof)
 * [Example 31](https://w3c.github.io/vc-data-model/#example-verifiable-presentation-using-the-data-integrity-bbs-cryptosuite-with-a-derived-credential-and-proof)
 * [Step 2](https://w3c.github.io/vc-data-model/#media-type-precision)

### PR DESCRIPTION
**Pull request by filip26**
This PR fixes issue #1556

***
<a href="https://pr-preview.s3.amazonaws.com/filip26/vc-data-model/pull/1557.html" title="Last updated on Aug 29, 2024, 3:59 PM UTC (c63b3c0)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1557/363a923...filip26:c63b3c0.html" title="Last updated on Aug 29, 2024, 3:59 PM UTC (c63b3c0)">Diff</a>

### CODE REVIEW

**Review by decentralgabe — APPROVED** [review]

**Review by dlongley — APPROVED** [review]

**Review by selfissued — APPROVED** [review]

**Review by iherman — APPROVED** [review]

**Review by brentzundel — APPROVED** [review]

**Review by msporny — APPROVED** [review]

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2024-09-11-vcwg#section3-1) on 2024-09-11

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 3.1. Fix #1556 (pr vc-data-model#1557)

_See github pull request [vc-data-model#1557](https://github.com/w3c/vc-data-model/pull/1557)._




**Brent Zundel:** going to merge this one after the call, lots of approvals, has been open for a long time.  
… minor but necessary changes.  
… next is 1554.  

</details>

**Comment by msporny** [comment]
Editorial, multiple reviews, no changes requested, no objections, merging.

**Pull request merged**
